<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AtomDesigner</class>
 <widget class="QWidget" name="AtomDesigner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>312</width>
    <height>453</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Atom</string>
  </property>
  <property name="toolTip">
   <string/>
  </property>
  <layout class="QVBoxLayout" name="_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>3</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Basic</string>
      </attribute>
      <layout class="QVBoxLayout" name="_3">
       <item>
        <widget class="QGroupBox" name="generalGB">
         <property name="title">
          <string>General</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="flagsGB">
         <property name="title">
          <string>Flags</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="specificGB">
         <property name="title">
          <string>Specific</string>
         </property>
         <layout class="QGridLayout" name="_4">
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Residue:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label4">
            <property name="text">
             <string>Symbol:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="symbolPB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="styleSheet">
             <string notr="true"> QPushButton {
     border: 1px none #dadbde;
     background-color: #FFFF7F;
     border-radius: 2px;
 }

 QPushButton:pressed {
     border: 2px solid #AAAAAA;
 }</string>
            </property>
            <property name="text">
             <string/>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Type:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="typeLE"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Serial index:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QSpinBox" name="serialIndexSB">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum">
             <number>10000000</number>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Local Index:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QSpinBox" name="localIndexSB">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="maximum">
             <number>10000000</number>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Charge:</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QQuantitySpinBox" name="chargeSB">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="minimum">
             <double>-10.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label1">
            <property name="text">
             <string>Structure:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QObjectLinkWidget" name="structureW" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QObjectLinkWidget" name="residueW" native="true">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="geometryTab">
      <attribute name="title">
       <string>Geometry</string>
      </attribute>
      <layout class="QVBoxLayout" name="_8">
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Position</string>
         </property>
         <layout class="QGridLayout" name="_5">
          <item row="2" column="2">
           <widget class="QQuantitySpinBox" name="xSB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-10000000000000000905969664.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000000000000000905969664.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLabel" name="label6">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QQuantitySpinBox" name="ySB">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-10000000000000000905969664.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000000000000000905969664.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLabel" name="label7">
            <property name="text">
             <string>Z:</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QQuantitySpinBox" name="zSB">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-10000000000000000905969664.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000000000000000905969664.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLabel" name="label5">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QCheckBox" name="moveWholeMolChBox">
            <property name="text">
             <string>Move whole molecule</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Velocity</string>
         </property>
         <layout class="QGridLayout" name="_6">
          <item row="0" column="0">
           <widget class="QLabel" name="label5_2">
            <property name="text">
             <string>X:</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QQuantitySpinBox" name="vxSB">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="prefix">
             <string/>
            </property>
            <property name="suffix">
             <string/>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-10000000000000000905969664.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000000000000000905969664.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label6_2">
            <property name="text">
             <string>Y:</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QQuantitySpinBox" name="vySB">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-10000000000000000905969664.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000000000000000905969664.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label7_2">
            <property name="text">
             <string>Z:</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QQuantitySpinBox" name="vzSB">
            <property name="alignment">
             <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
            </property>
            <property name="decimals">
             <number>4</number>
            </property>
            <property name="minimum">
             <double>-10000000000000000905969664.000000000000000</double>
            </property>
            <property name="maximum">
             <double>10000000000000000905969664.000000000000000</double>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Bonds</string>
      </attribute>
      <layout class="QVBoxLayout" name="_9">
       <item>
        <widget class="QTreeView" name="bondsTV">
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="rootIsDecorated">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="_7">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Number of bonds:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="numOfBondsLE">
           <property name="frame">
            <bool>false</bool>
           </property>
           <property name="alignment">
            <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
           </property>
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="refByTab">
      <attribute name="title">
       <string>Referenced by</string>
      </attribute>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Close|QDialogButtonBox::Reset</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QObjectLinkWidget</class>
   <extends>QWidget</extends>
   <header location="global">ObjectLinkWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QQuantitySpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header location="global">QuantitySpinBox.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
